cloud_governance/common/clouds/azure/cost_management/cost_management_operations.py

```diff
@@ -80,6 +80,11 @@ def get_forecast(self, scope: str, start_date: datetime = '', end_date: datetime
                     row_data[data_date] = data
             result['rows'] = list(row_data.values())
             return result
+        except HttpResponseError as e:
+            logger.error(e)
+            if e.status_code == 429:
```

Comment:
Remove this if

Reply:
No, we need to handle only 429. Others we need to raise error